### PR TITLE
Adds wemake-vue-template

### PR DESCRIPTION
**Pull request by sobolevn**
Source code: https://github.com/wemake-services/wemake-vue-template/tree/master/template
Demo link: https://wemake-vue-demo.herokuapp.com/

### For all PRs

- [x] I read the [Contribution Guide](CONTRIBUTING.md)

### For new stacks

- [x] I followed the [do's and don'ts](CONTRIBUTING.md#contributing-dos-and-donts)
- [x] This stack does not already exist

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -122,6 +122,27 @@ A Hugo boilerplate for creating truly epic websites. This is a boilerplate for u
 
 - [netlify-templates/victor-hugo](https://github.com/netlify-templates/victor-hugo)
 
+
+## Nuxt, TypeScript, Docker [↗](https://awesomestacks.dev/wemake-vue-template)
```
Comment (jdorfman) [inline README.md]: The uri doesn't match `/wemake-vue-template`

File: README.md [hunk 2]
```diff
@@ -122,6 +122,27 @@ A Hugo boilerplate for creating truly epic websites. This is a boilerplate for u
...
+
+Bleeding edge vue template focused on code quality and developer happiness.
+
+Offers a complete setup for both small and enterprise-scale apps.
+
+- [Nuxt](https://nuxtjs.org/) - [🛠](https://stackshare.io/nuxt)- [🐙](https://github.com/nuxt/nuxtjs) - Vue.js Meta Framework to create complex, fast & universal web applications quickly.
```
Comment (jdorfman) [inline README.md]: The url "https://github.com/nuxt/nuxtjs" is broken

File: README.md (hunk 1 above)
Comment (jdorfman) [reply README.md]: Please update title to `Nuxt.js, TypeScript + Docker`

uri to: `/nuxtjs-typeScript-docker`

**Comment by sobolevn** [comment]
Done! Thanks for your review.

**Comment by sobolevn** [comment]
Copy-pasted your link ( https://github.com/nuxt/nuxt.js) - it is identical to mine. Hope it will work this time 🙂

**Comment by jdorfman** [comment]
Awesome thanks @sobolevn 

Feel free to link back to our repo in your readme ;)

**Comment by sobolevn** [comment]
Will do!

**Pull request merged**